--- dllama_orig/distributed-llama/src/app.cpp	2026-05-16 13:23:14
+++ dllama_patched/src/app.cpp	2026-05-15 16:29:44
@@ -124,6 +124,8 @@
             args.gpuSegmentTo = atoi(separator + 1);
         } else if (std::strcmp(name, "--net-turbo") == 0) {
             args.netTurbo = atoi(value) == 1;
+        } else if (std::strcmp(name, "--nbatches") == 0) {
+            args.nBatches = (NnUint)atoi(value);
         } else {
             throw std::runtime_error("Unknown option: " + std::string(name));
         }
